File: specification/specs/analysis/token_filters/CommonGramsTokenFilter.ts
class CommonGramsTokenFilter extends TokenFilterBase {
	@prop_serializer("SingleOrEnumerableFormatter`1")
	common_words: string[];
	common_words_path: string;
	@prop_serializer("NullableStringBooleanFormatter")
	ignore_case: boolean;
	@prop_serializer("NullableStringBooleanFormatter")
	query_mode: boolean;
}

File: output/csharp/Analysis/TokenFilters/CommonGramsTokenFilter.cs
using Nest.Analysis;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Analysis {

	public class CommonGramsTokenFilter : TokenFilterBase {
		
		[DataMember(Name="common_words")]
		public List<string> CommonWords { get; set; }


		[DataMember(Name="common_words_path")]
		public string CommonWordsPath { get; set; }


		[DataMember(Name="ignore_case")]
		public bool IgnoreCase { get; set; }


		[DataMember(Name="query_mode")]
		public bool QueryMode { get; set; }

	}
}
